app/models/taxon.rb

```diff
@@ -68,4 +74,11 @@ def can_subscribe?
 
     true
   end
+
+private
+
+  def fetch_tagged_content
+    taxon_content_ids = [content_id] + associated_taxons.map(&:content_id)
+    TaggedContent.fetch(*taxon_content_ids)
```

Comment:
What does the splat do here? Does it turn the array into individual multiple arguments that get collected by the receiver's catch all?